arviz/plots/backends/matplotlib/traceplot.py

```diff
@@ -210,57 +220,26 @@ def plot_trace(
                     invalid_var_names, all_var_names
                 )
             )
-
+    # pylint: disable=too-many-nested-blocks
     for idx, (var_name, selection, value) in enumerate(plotters):
-        value = np.atleast_2d(value)
+        for idy in range(2):
```

Comment:
The idy variable is introduced because axes can have only one projection so we can not mix polar and regular axes, right? So we need to create and access them as separated objects, right?

Reply:
Yes, that is right